src/frontend/src/utils/condition.rs

```diff
@@ -74,49 +74,29 @@ impl Condition {
     }
 
     pub fn to_expr(self) -> ExprImpl {
```

Comment:
Suggest naming as `into_expr`.

Reply:
Then why not `impl Into<ExprImpl> for Condition`